### PR TITLE
feat(protocol-designer): add wrap to protocol overview columns

### PR DESCRIPTION
**Pull request by ncdiehl11**
# Overview

Wrap second `ProtocolOverview` column once minimum page width is hit

Closes AUTH-716

## Test Plan and Hands on Testing

Create a protocol and resize `ProtocolOverview` page. Verify that column break occurs when page width hits 1184px.
https://github.com/user-attachments/assets/38f20fbc-0d11-4d76-a3f0-f64f3bce2180

## Changelog

- add line break to `ProtocolOverview`
- add optional width prop to `DeckThumbnail` and `OffdeckThumbnail`
- fix color picker bug while in there

## Risk assessment

low

### CODE REVIEW

File: protocol-designer/src/pages/ProtocolOverview/DeckThumbnail.tsx [hunk 1]
```diff
@@ -98,7 +99,7 @@ export function DeckThumbnail(props: DeckThumbnailProps): JSX.Element {
   return (
     <Flex
       height="404px"
-      width="520px"
+      width={width}
```
Comment (jerader) [inline protocol-designer/src/pages/ProtocolOverview/DeckThumbnail.tsx]: i think there is a way to set the width without needing to add a prop. Using `maxWidth = 32.5rem` and `width: 100%`, i think might do the trick?

**Review by jerader — APPROVED** [review]
looks great! left a suggestion but otherwise lgtm

File: protocol-designer/src/pages/ProtocolOverview/DeckThumbnail.tsx [hunk 2]
```diff
@@ -98,7 +99,7 @@ export function DeckThumbnail(props: DeckThumbnailProps): JSX.Element {
   return (
     <Flex
       height="404px"
```
Comment (koji) [inline protocol-designer/src/pages/ProtocolOverview/DeckThumbnail.tsx]:
```suggestion
      height="25.25rem"
```

**Comment by koji** [comment]
@ncdiehl11 
could you resolve the conflicts and merge this PR into `edge` when you are available?

**Pull request merged**